src/mono/wasm/templates/templates/browser/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+[assembly:System.Runtime.Versioning.SupportedOSPlatform("browser")]
```

Comment:
Is this the best approach we can offer?

Reply:
IIUC, this is needed. I'm not sure how else we can do this, but open to suggestions!